### PR TITLE
Data release 20240714

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json [hunk 1]
```diff
@@ -4906,33 +4896,6 @@
         "Count_Household_WithFoodStampsInThePast12Months_ForeignBorn_PlaceOfBirthSouthamerica",
         "Count_Household_WithFoodStampsInThePast12Months_ForeignBorn_PlaceOfBirthSouthernEasternEurope",
         "Count_Household_WithFoodStampsInThePast12Months_ForeignBorn_PlaceOfBirthWesternAsia",
-        "Count_Household_WithFoodStampsInThePast12Months_HispanicOrLatino",
```
Comment (hareesh-ms) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:4909]: S2201 refresh (cl/648291018) seems to have removed certain geo levels. Following up on this.

File: internal/server/statvar/golden/search_statvar/count_person.json [hunk 2]
```diff
@@ -3919,86 +3999,6 @@
     {
       "name": "Population: 1 - 4 Years, Different House in Different County Different State",
       "dcid": "Count_Person_1To4Years_DifferentHouseInDifferentCountyDifferentState"
-    },
```
Comment (hareesh-ms) [inline internal/server/statvar/golden/search_statvar/count_person.json:3922]: Some of the new variables were added and existing removed from the 1000 SV list.

File: internal/server/statvar/golden/search_statvar/fem.json [hunk 3]
```diff
@@ -748,22 +764,6 @@
       "name": "Proportion of vulnerable population receiving social assistance cash benefit [Female]",
       "dcid": "sdg/SI_COV_VULN.SEX--F"
     },
-    {
```
Comment (hareesh-ms) [inline internal/server/statvar/golden/search_statvar/fem.json:751]: Related to cl/647778136 

@n-h-diaz  - Please confirm these deletions were expected

**Review by ajaits — APPROVED** [review]
New Svs expected from NFHS, placs from boundary imports. SGD deletions from cl/647778136

File: internal/server/statvar/golden/search_statvar/fem.json (hunk 3 above)
Comment (n-h-diaz) [reply internal/server/statvar/golden/search_statvar/fem.json:751]: The deletions are expected in the refresh

**Review by n-h-diaz — APPROVED** [review]

**Pull request merged**